spectrum/src/main/java/io/github/giulong/spectrum/utils/JsMethodsUtils.java

```diff
@@ -0,0 +1,75 @@
+package io.github.giulong.spectrum.utils;
+
+import io.github.giulong.spectrum.enums.LocatorType;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.*;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@Getter
+@NoArgsConstructor(access = PRIVATE)
+public final class JsMethodsUtils {
```

Comment:
I did all the refactoring of this class, in conclusion it has been deleted because the logic is now managed by the LocatorType Enum. 

I do not resolve the conversation for you to check and leave me a feedback 👍 

commit 5665e0173786a045bd39b12619d029e5890561d1